host/tsconfig.json

```diff
@@ -2,11 +2,5 @@
   "extends": "../tsconfig.base.json",
   "outDir": "./dist",
   "rootDir": "./src",
-  "strict": false,
-  "compilerOptions": {
-    "declaration": true,
-    "declarationDir": "./types",
-    "declarationMap": true,
-    "emitDeclarationOnly": true
-  }
+  "strict": false
```

Comment:
dunno what im doing here, it was just generating a lot of files.. 🤷

Reply:
I suspect we don't need the defs. and just to test the hook and prove the point.. bfcd849